plugins/newsdownloader.koplugin/main.lua

```diff
@@ -105,6 +106,21 @@ function NewsDownloader:addToMainMenu(menu_items)
                 keep_menu_open = true,
                 callback = function() self:removeNewsButKeepFeedConfig() end,
             },
+            {
+                text = _("Force skip images download"),
+                keep_menu_open = true,
+                callback = function()
+                    local news_downloader_settings = LuaSettings:open(("%s/%s"):format(DataStorage:getSettingsDir(), news_downloader_config_file))
+                    local skip = news_downloader_settings:readSetting(force_skip_images_download_config_name) or false;
+                    logger.warn('NewsDownloader: previous force_skip_images_download: ', skip)
```

Comment:
```suggestion
                    logger.info("NewsDownloader: previous force_skip_images_download: ", skip)
```

Reply:
But that whole never/don't stuff isn't necessary and harder to parse, better yet is simply reversed logic:

![Screenshot_20200326_085725](https://user-images.githubusercontent.com/202757/77623223-02930100-6f40-11ea-9917-4ea0b071f820.png)

Reply:
The hole idea was to make it a quick way to sometimes disable images (e.g.  slow wifi, no time for downloading all data, use smaller amount of wifi in case of being currently connected to limited transfer wifi etc). 

I didn't want to make it a replacement for" include_images " which can be set for each rss/atom source independly.

And  the above screenshot looks this way for me that 's why I wanted to mark it as "skip" / "temporally" etc..

Reply:
Btw I like the idea with checkbox (and would like to have it) but remembered that moving "lua Settings open" to lazy load was because there was a need/request to speed up menu. 
That's why I didn't use it..

Reply:
Maybe you're thinking of something wifi-related? It already says `self:lazyInitialization()` in addToMainMenu so regardless how slow that may or may not be, the cost is incurred.

https://github.com/koreader/koreader/blob/0b898624acc8b332d44a16c95b45f68f4ccb1297/plugins/newsdownloader.koplugin/main.lua#L72-L73

Reply:
Ok, how about checkbox + name that indicates enabling won't cause automatically downloading images from all sources (as it still has to be enabled in feed_config.lua)?

Reply:
I have no problem with `Never download images`.

Reply:
Ok checkbox + "never download images " is much safer will do so. 

BTW I'm not familiar with recent git changes. 
Although I haven't touch the majority of the PR request changes they are marked as 'outdated'. 
Do I need to changed them or you somehow patched it?

Reply:
No, that just means I thought you submitted a PR with 4 distinct commits which I mistakenly reviewed independently.